ui/widgets/tabs.js

```diff
@@ -312,10 +312,6 @@ $.widget( "ui.tabs", {
 		}
 	},
 
-	_sanitizeSelector: function( hash ) {
-		return hash ? hash.replace( /[!"$%&'()*+,.\/:;<=>?@\[\]\^`{|}~]/g, "\\$&" ) : "";
-	},
```

Comment:
I see there's at least one case of `$.fn._focus` getting renamed to `$.fn.focus` between `1.10.1` & `1.10.2`: https://github.com/jquery/jquery-ui/compare/1.10.1...1.10.2

Reply:
Ohh that is in indeed a good question. Iirc Scott would not make a big thing of changing internal methods but I'm not really sure. 

Why not just keep the `$.fn._sanitizeSelector` until the next minor release but make it use `CSS.escape`?

Reply:
Just seen its not `$.fn` but `$.ui.tabs` so it's even more specific (as in Tabs widget only). Not sure if its worth keeping the old method around...

Reply:
OK, I'll keep it removed then. There's no universal way to sanitize the whole selector; the safe thing to do is to just escape the identifier part, especially when potentially coming from user input. Even if I changed the internal method to use `CSS.escape`, it would just handle the "# + id" case, which can be confusing and can still lead people to use it where it's not intended.